lib/pacio_test_kit/pfe/single_observation_group.rb

```diff
@@ -1,4 +1,5 @@
 require_relative '../common_tests/read_test'
+require_relative '../common_tests/create_test'
```

Comment:
only need  to add create test to one observation profile.  It was already added to the clinical test profile